src/qt/walletcontroller.cpp

```diff
@@ -231,7 +232,11 @@ void CreateWalletActivity::askPassphrase()
 
 void CreateWalletActivity::createWallet()
 {
-    showProgressDialog(tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
+    showProgressDialog(
+        //: Title of progress window when Create Wallet action is selected.
+        tr("Create Wallet"),
+        //: Describing what is being done in background.
+        tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
```

Comment:
while here, can we break up the HTML tags from the string to be translated? Keeping these tags as part of the string is error-prone

```diff
@@ -236,7 +236,7 @@ void CreateWalletActivity::createWallet()
         //: Title of progress window when Create Wallet action is selected.
         tr("Create Wallet"),
         //: Describing what is being done in background.
-        tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
+        tr("Creating Wallet") + QString(" <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
 
     std::string name = m_create_wallet_dialog->walletName().toStdString();
     uint64_t flags = 0;
@@ -331,7 +331,7 @@ void OpenWalletActivity::open(const std::string& path)
         //: Title of progress window when Open Wallet action is selected.
         tr("Open Wallet"),
         //: Describing what is being done in the background.
-        tr("Opening Wallet <b>%1</b>…").arg(name.toHtmlEscaped()));
+        tr("Opening Wallet") + QString(" <b>%1</b>…").arg(name.toHtmlEscaped()));
 
     QTimer::singleShot(0, worker(), [this, path] {
         std::unique_ptr<interfaces::Wallet> wallet = node().walletClient().loadWallet(path, m_error_message, m_warning_message);
```

Reply:
> Keeping these tags as part of the string is error-prone

Oh, I was not aware of it. Thanks for enlightening me! Addressed in ccecfa8

Reply:
> while here, can we break up the HTML tags from the string to be translated? Keeping these tags as part of the string is error-prone

Why?

![Screenshot from 2021-10-04 22-56-56](https://user-images.githubusercontent.com/32963518/135916075-502c09bc-32c6-432b-aece-8b120cc6054a.png)